src/m365/spo/commands/navigation/navigation-node-add.ts

```diff
@@ -126,6 +127,11 @@ class SpoNavigationNodeAddCommand extends SpoCommand {
       logger.logToStderr(`Adding navigation node...`);
     }
 
+    let url = args.options.url;
+    if (!url) {
+      url = 'http://linkless.header/';
+    }
```

Comment:
Let's use `??` to shorten this to one line. According to me we can assign it directly to the request body property then.